_spinnaker/operator.md

```diff
@@ -17,36 +20,55 @@ Spinnaker Operator is a Kubernetes operator for Spinnaker that makes it easier t
 Operator has two distinct modes you can install and use:
 
 - **Basic**: Operator in basic mode installs Spinnaker into a single namespace without `ValidatingAdmissionWebhook` for doing preflight checks.
-- **Cluster**: Operator in cluster mode installs Spinnaker across multiple namespaces with `ValidatingAdmissionWebhook` for doing preflight checks. This mode requires a `ClusterRole`.
+- **Cluster**: Operator in cluster mode installs Spinnaker across namespaces with `ValidatingAdmissionWebhook` for doing preflight checks. This mode requires a `ClusterRole`.
+
+# Benefits of Operator
+
+- Stop using Halyard commands: just `kubectl apply` your Spinnaker configuration. This includes support for local files.
+- Expose Spinnaker to the outside world (via `LoadBalancer`). You can still disable that behavior if you prefer to manage ingresses and load balancers yourself. 
+- Deploy any version of Spinnaker. Operator is not tied to a particular version of Spinnaker. 
+- Keep secrets separate from your config. Store your config in `git` and have an easy Gitops workflow.
+- Validate your configuration before applying it (with webhook validation). 
+- Store Spinnaker secrets in Kubernetes secrets.
+- Patch versions, accounts or any setting with `kustomize`. 
+- Monitor the health of Spinnaker via `kubectl`.
+- Define Kubernetes accounts in `SpinnakerAccount` objects and store kubeconfig inline, in Kubernetes secrets, in s3, or GCS **(Experimental)**.
 
```

Comment:
Before jumping into the nitty gritty, I'd add the "Spinnaker installed in under a minute (or two)" section. It's really all there is to the operator and it's very clear. Feel free to rename the section to something less catchy but we need an overview.